xarray/tests/test_variable.py

```diff
@@ -2540,3 +2542,68 @@ def test_clip(var):
             var.mean("z").data[:, :, np.newaxis],
         ),
     )
+
+
+@pytest.mark.parametrize("Var", [Variable, IndexVariable])
```

Comment:
you could stick these in `VariableSubclassObjects`? Could also do that later if it's a good idea

Reply:
I just had a go at that and it broke the pint-only test through interaction with `TestVariableWithDask`, which tries to chunk.